pkg/exporter_test.go

```diff
@@ -0,0 +1,598 @@
+package exporter
```

Comment:
It's probably easier to review if land https://github.com/prometheus-community/yet-another-cloudwatch-exporter/pull/1791 first.

Reply:
This is a good idea for the regression testing in the first place, and we can reuse some mocks.

We can review and merge it. However, let's not block this one because of it.

cc: @tristanburgess